app/javascript/packs/internal.tsx

```diff
@@ -1,6 +1,6 @@
 import 'easymde/dist/easymde.min.css'
 
-require('channels')
+import Channels from 'channels'
```

Comment:
What does this do? We don't seem to use `Channels` anywhere in here. Did requiring do something different in the past (like init something?)

Reply:
Possibly. I'll see if removing breaks anything